web/src/stream_create.js

```diff
@@ -58,15 +61,16 @@ class StreamSubscriptionError {
 const stream_subscription_error = new StreamSubscriptionError();
 
 class StreamNameError {
-    report_already_exists() {
-        $("#stream_name_error").text(
-            $t({defaultMessage: "A stream with this name already exists"}),
-        );
+    report_already_exists(error) {
+        const errorMessage =
```

Comment:
Use `error_message` -- we don't use camelCase variable names.

Reply:
Yeah I was aware of that but looking at `defaultMessage` I thought in this specific instance, I needed to use camelCase.
Fixed though.